### ISSUE TITLE
Can't `ls/lsd` contents with Storj, but it works in S3 mode

### ISSUE THREAD

**Issue by acsfer** [rclone/rclone#5795]
#### What is the problem you are having with rclone?

I can list contents of Storj bucket in S3 mode, but not with native Storj credentials.

#### What is your rclone version (output from `rclone version`)

```
rclone v1.57.0
- os/version: ubuntu 20.04 (64 bit)
- os/kernel: 5.4.0-74-generic (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.17.2
- go/linking: static
- go/tags: none
```

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

Ubuntu 20.04.3 LTS

#### Which cloud storage system are you using? (e.g. Google Drive)

Storj

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

```
# rclone lsd Storj:backups -vv
2021/11/08 17:57:24 DEBUG : rclone: Version "v1.57.0" starting with parameters ["rclone" "lsd" "Storj:backups" "-vv"]
2021/11/08 17:57:24 DEBUG : Creating backend with remote "Storj:backups"
2021/11/08 17:57:24 DEBUG : Using config file from "/root/.config/rclone/rclone.conf"
2021/11/08 17:57:24 DEBUG : FS sj://backups: connecting...
2021/11/08 17:57:24 DEBUG : FS sj://backups: connected: <nil>
2021/11/08 17:57:24 DEBUG : FS sj://backups: ls ./
2021/11/08 17:57:24 DEBUG : FS sj://backups: OBJ ls ./ ("backups", "")
2021/11/08 17:57:24 DEBUG : FS sj://backups: opts &{Prefix: Cursor: Recursive:false System:true Custom:true}
2021/11/08 17:57:45 DEBUG : 5 go routines active
```

vs

```
# rclone lsd StorjS3:backups -vv
2021/11/08 17:58:19 DEBUG : rclone: Version "v1.57.0" starting with parameters ["rclone" "lsd" "StorjS3:backups" "-vv"]
2021/11/08 17:58:19 DEBUG : Creating backend with remote "StorjS3:backups"
2021/11/08 17:58:19 DEBUG : Using config file from "/root/.config/rclone/rclone.conf"
           0 2021-11-08 17:58:50        -1 data
           0 2021-11-08 17:58:50        -1 index
           0 2021-11-08 17:58:50        -1 keys
           0 2021-11-08 17:58:50        -1 snapshots
2021/11/08 17:58:50 DEBUG : 3 go routines active
```

### Config file (redacted)

```
[Storj]
type = tardigrade
access_grant = 1cCYAHiNyogwPRkZbkpo8C6txEeJ4cktWZoHzwxxxxxxxxxxxxxxx

[StorjS3]
type = s3
provider = Other
access_key_id = jxvq7tmxxxxxxxxxxxxxx
secret_access_key = jzqr7r3kgkglljd3icpiexxxxxxxxxxxxxxxxxxxxxxxxx
endpoint = gateway.eu1.storjshare.io
acl = private
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
The listing looks like it didn't return an error.

Are you sure the two things are listing the same place/account?

Can you create a bucket / upload stuff with the native protocol?

**Comment by acsfer** [comment]
I figured it out; when you create Storj credentials, is asks for a passphrase.
Obviously, if you enter different ones for each credentials, buckets look empty (because hey are encrypted and can't be decrypted).

TL;DR enter the same passphrase for each credentials, Uplink or S3.
